### PR TITLE
the one that updates blockquote to allow for use with context and a bit more

### PR DESCRIPTION
**Pull request by sturobson**
This includes:

- an update to allow for using `vf-blockquote` as an 'include' in VF Eleventy sites
- includes the option for a `cite` element to help make block quote be a little more usable. 
- makes use of `vf-stack` for vertical layout
- lets `cite` just 'do it's thing' from the browser agent and just apply `font-style: italic` to the element.

### CODE REVIEW

File: components/vf-blockquote/vf-blockquote.njk
Comment (khawkins98) [inline components/vf-blockquote/vf-blockquote.njk:2]: could we add a fallback to use `html` or `text` if `blockquote_text` is not set? (stops it from being a breaking change)

Comment (sturobson) [reply components/vf-blockquote/vf-blockquote.njk:2]: I'll take a look

Comment (sturobson) [reply components/vf-blockquote/vf-blockquote.njk:2]: it's a tongue twister but
```
{% if (blockquote_text) or (html) or (text) %}
    <p class="vf-blockquote__text">{{- (html) or (blockquote_text) | safe if (html) or (blockquote_text) else text -}}</p>
  {% endif %}
```

works for this

Comment (sturobson) [reply components/vf-blockquote/vf-blockquote.njk:2]: @khawkins98 I'll let you resolve this and I'll merge when approved

**Review by khawkins98 — APPROVED** [review]

**Pull request merged**
